dts/bindings/vendor-prefixes.txt

```diff
@@ -306,6 +306,7 @@ okaya	Okaya Electric America, Inc.
 oki	Oki Electric Industry Co., Ltd.
 olimex	OLIMEX Ltd.
 olpc	One Laptop Per Child
+omnivision OmniVision Technologies Co., Ltd.
```

Comment:
Under Linux OmniVision Technologies vendor prefix is 'ovti' , I would prefer that one for consistency.

Reply:
Very reasonable. Fixed